bootstrap.py

```
@@ -238,11 +239,12 @@ def _init_config(
 
 
 def _register_bot_client_provider(context: Context) -> None:
-    """注册 bot client provider，供主动推送场景下的 NapCat stream 使用。
+    """注册 bot client / bot self_id provider，供主动推送场景使用。
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)